src/app/calculator/utilities/cash-flow/cash-flow.service.ts

```diff
@@ -207,15 +212,19 @@ export class CashFlowService {
     let sumContinueB: number = 0;
     let sumIterationB: number = 0;
     bruteForceResults.forEach(result => {
-      sumContinueA += result.continueA;
+      //sumContinueA += result.continueA;
```

Comment:
Block from 214 on: 
1.NaNs can likely be prevented earlier on so you don't need the utility method. I would guess they're being introduced where you're incrementing in this loop.
2.There is redundancy re: iteration/loops in ln 221 and on. If we only need the continuelists to sum them up, JS array methods like reduce() could be used to make things clearer.